### DISCUSSION TITLE
Add a all coverages overlay

### DISCUSSION THREAD

**Discussion by LDprg** [OpenTTD/OpenTTD#12756] [Features]
Everyone knows the situation where you click through every station and clicking every  coverage button for eye balling the city total coverage. Therefore this feature request is about a button for viewing all coverages at the same time, so it is very easy to see the missing spots.

Optionally this could aso include a per city coverage in percent, calculated by checking the amount of buildings within a coverage area. Just a handy statistics, however not as important as the possibility to view all coverage areas at the same time.

This shouldn't be too hard to implement since it doesn't invent anything new. I have read somewhere that a patch for this does exists somewhere in the Internet, but I wasn't able to find it.

This is somewhat related to #7446

**Comment by LC-Zorg** [comment]
The function you are writing about has been available in the game for several years (1.10.0). ;)
![zooning - regular game](https://github.com/OpenTTD/OpenTTD/assets/67662046/2ba6a825-b519-48ea-89f1-28a4a9221d7a)
As you probably guessed, buildings outside the station's range are marked in red.

Moreover, there may be a dozen or so covers in total. Displaying "all" of them at the same time would be quite risky. :D The JGRPP and CityMania versions of the game have the ability to show one or two different ranges together. Below is a screenshot of the CityMania version that I think does it best.
![zooning - citymania version](https://github.com/OpenTTD/OpenTTD/assets/67662046/f7216718-b268-4b13-90c6-0f014d3bcea3)

**Reply by LDprg** [reply]
@LC-Zorg Thx, how could I missed the coverage button in the city menu! Maybe there should be a notice about it in the wiki.
